install.sh

```diff
@@ -404,18 +406,32 @@ install_airgap_tarball() {
 # do_install_rpm builds a yum repo config from the channel and version to be installed,
 # and calls yum to install the required packates.
 do_install_rpm() {
-    maj_ver="7"
-    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
-        dist_version="$(. /etc/os-release && echo "$VERSION_ID")"
-        maj_ver=$(echo "$dist_version" | sed -E -e "s/^([0-9]+)\.?[0-9]*$/\1/")
-        case ${maj_ver} in
-            7|8)
-                :
-                ;;
-            *) # In certain cases, like installing on Fedora, maj_ver will end up being something that is not 7 or 8
-                maj_ver="7"
-                ;;
-        esac
+    . /etc/os-release
+    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ] || [ "${ID_LIKE%%[ ]*}" = "suse"  ]; then
+        repodir=/etc/yum.repos.d
+        if [ -d /etc/zypp/repos.d ]; then
+            repodir=/etc/zypp/repos.d
+        fi
+        if [ "${ID_LIKE%%[ ]*}" = "suse" ]; then
```

Comment:
The `"${ID_LIKE%%[ ]*}`  here looks like a bash-ism, does it work on busybox /bin/sh?

Reply:
I am not sure, but I guess not, but do we support installation on busybox anyway?

Reply:
I'd vote to make everything as POSIX compatible as possible.

Reply:
We need to support basic Bourne shell syntax since we run the installer with /bin/sh; this means no bash-isms.

Reply:
makes sense, I will check for compatibility but FWIW we use the same syntax in k3s install script https://github.com/k3s-io/k3s/blob/master/install.sh#L463

Reply:
I was mostly concerned about the bracket expansion; if it works in K3s it should work here too!
```console
/ # ID_LIKE="suse foo bar"; echo ${ID_LIKE%%[ ]*}
suse
```

Reply:
yeah I tested it and its  working as expected in busybox /bin/sh